### ISSUE TITLE
[BUG] Crash on dedicated server start with spectrum-1.1.5-1.18.1-earlygame6.jar

### ISSUE THREAD

**Issue by natebc** [DaFuqs/Spectrum#12]
When starting a dedicated server (modpack: Benchmark-II, Version 0.4.6, issue reported: https://github.com/Chuunkyz/BenchmarkII-1.18/issues/22)

To reproduce I installed the modpack  clean and got the same crash.

Happy to help troubleshoot in any way I can. Please let me know if there's any information you need.

latest.log from the clean start to crash:
https://gist.github.com/natebc/002206f114b370bf0b332bc46b95c1dd

stacktrace:
```
[13:11:53] [main/FATAL]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'spectrum'!
        at Not Enough Crashes deobfuscated stack trace.(1.18.1+build.12) ~[?:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:62) ~[fabric-loader-0.12.12.jar:?]
        at net.minecraft.server.Main.main(Main:106) [server-intermediary.jar:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) [fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.12.12.jar:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:66) [fabric-loader-0.12.12.jar:?]
Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_437
        at de.dafuqs.spectrum.registries.SpectrumItemGroups$1.<init>(SpectrumItemGroups.java:21) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumItemGroups.<clinit>(SpectrumItemGroups.java:20) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumBlocks.<clinit>(SpectrumBlocks.java:87) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.SpectrumCommon.onInitialize(SpectrumCommon.java:88) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.12.12.jar:?]
        ... 15 more
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_437
        at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
        at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
        at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:158) ~[fabric-loader-0.12.12.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
        at de.dafuqs.spectrum.registries.SpectrumItemGroups$1.<init>(SpectrumItemGroups.java:21) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumItemGroups.<clinit>(SpectrumItemGroups.java:20) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumBlocks.<clinit>(SpectrumBlocks.java:87) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.SpectrumCommon.onInitialize(SpectrumCommon.java:88) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.12.12.jar:?]
        ... 15 more
```

**Comment by TheRanker** [comment]
also have the same issue..

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'spectrum'!
        at Not Enough Crashes deobfuscated stack trace.(1.18.1+build.12) ~[?:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:62) ~[fabric-loader-0.12.12.jar:?]
        at net.minecraft.server.Main.main(Main:106) [server-intermediary.jar:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) [fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.12.12.jar:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:66) [fabric-loader-0.12.12.jar:?]
Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_437
        at de.dafuqs.spectrum.registries.SpectrumItemGroups$1.<init>(SpectrumItemGroups.java:21) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumItemGroups.<clinit>(SpectrumItemGroups.java:20) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumBlocks.<clinit>(SpectrumBlocks.java:87) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.SpectrumCommon.onInitialize(SpectrumCommon.java:88) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.12.12.jar:?]
        ... 15 more
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_437
        at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
        at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
        at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:158) ~[fabric-loader-0.12.12.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
        at de.dafuqs.spectrum.registries.SpectrumItemGroups$1.<init>(SpectrumItemGroups.java:21) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumItemGroups.<clinit>(SpectrumItemGroups.java:20) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumBlocks.<clinit>(SpectrumBlocks.java:87) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.SpectrumCommon.onInitialize(SpectrumCommon.java:88) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.12.12.jar:?]
        ... 15 more

**Comment by DaFuqs** [comment]
Are you using the newest version of owo-lib?
It was fixed on that end, though I have not checked if the newest version actually fixes it.
See: https://github.com/glisco03/owo-lib/issues/9

**Comment by DaFuqs** [comment]
The upcoming version of Spectrum tomorrow will add a workaround for this.

**Comment by TheRanker** [comment]
I'm using latest owo yes. Still happens.

**Comment by natebc** [comment]
seems that the pack maker for Benchmark II has updated oωo in the last hour or so and this appears to have fixed my problem.

**Comment by DaFuqs** [comment]
I commented on the original owo-issue that the issue still persist. Thank you for the quick resonse.

**Comment by DaFuqs** [comment]
note: it is my fault. Leave glisco alone. :P

**Comment by DaFuqs** [comment]
Fixed in 1.2.0
